docs/general/testing/server.md

```diff
@@ -0,0 +1,54 @@
+---
+uid: testing-server
+title: Testing Jellyfin Server
+---
+
+# Testing Jellyfin Server
+
+:::caution Backups!
+
+Unstable often makes irreversible changes to existing Jellyfin setups. Please make regular backups of Jellyfin files. [Read More](/docs/general/administration/configuration#server-paths)
+
+:::
+
+## Weekly Unstable Build
+
+The weekly unstable builds are generated every Monday 5:00 AM UTC. Testing using the Unstable builds can not only test Jellyfin itself, but also help identify issues with packaging.
+
+To install them, navigate to our [downloads page](/downloads/server) and choose `Server`, choose your platform, then choose `Unstable`. The choices have to be made in this order. After choosing, scroll down and find the installation method you would like to use.
+
+![Downloads Page](/images/docs/testing/server/weekly-1.png)
+
+## Testing from Master Branch
+
+For the absolute bleeding edge, you can test directly from the Master branch of the server. To test from the master branch, a few extra tools are needed:
```

Comment:
I'd avoid mentioning this here, or at least, not in this way. Really only *developers* should be building off master like this, not normal users.

Reply:
Maybe move this to a different section of the docs and just leave a link here?